pkgs/by-name/ph/phel/package.nix

```diff
@@ -5,16 +5,16 @@
 
 php.buildComposerProject (finalAttrs: {
   pname = "phel";
-  version = "0.13.0";
+  version = "0.15.3";
```

Comment:
Try this one `0.16.0` 🚀  https://github.com/phel-lang/phel-lang/releases/tag/v0.16.0